Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -1,220 +1,234 @@
+    }
 
-        }
+    override func viewDidDisappear(_ animated: Bool) {
 
-        func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-            if packArray.isEmpty  {
-                return 1
+        super.viewDidDisappear(animated)
+        // When leaving this view controller, remove all offline downloads.
+        guard let packs = MGLOfflineStorage.shared.packs else { return }// or do something else }
+        MGLOfflineStorage.shared.removePack(packs[0]) { (error) in
+            // handle error if pack can't be removed
+            if let error = error {
+                print(error)
             } else {
-                return packArray.count
+                self.tableView.reloadData()
             }
-
+            //mechanisms to reload certain cells instead of entire tableview
         }
 
+    }
 
-        func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-            let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
-            if MGLOfflineStorage.shared.packs?.count != nil {
-                let progress = Float(packArray[indexPath.row].progress.countOfResourcesCompleted)/Float(packArray[indexPath.row].progress.countOfResourcesExpected)
-                cell.textLabel!.text = "Region \(indexPath.row + 1): \n  Percent completion: \(String(format: "%.2f", progress * 100)) %"
-                print(Float((packArray[indexPath.row].progress.countOfResourcesCompleted)/(packArray[indexPath.row].progress.countOfResourcesExpected) * 100))
-
-                cell.textLabel?.font = UIFont.systemFont(ofSize: 15)
-                cell.textLabel?.numberOfLines = 0
-                cell.textLabel?.lineBreakMode = .byWordWrapping
-                if #available(iOS 13.0, *) {
-                    cell.textLabel?.textColor = .darkGray
-                } else {
-                    // Fallback on earlier versions
-                }
-                cell.textLabel?.textAlignment = .left
+    deinit {
+        // Remove offline pack observers.
+        print("Removing offline pack notification observers")
+        NotificationCenter.default.removeObserver(self)
 
-            } else {
+    }
 
-                cell.textLabel?.text = "No Offline Packs Saved"
-                cell.textLabel?.textColor = .darkGray
-                cell.textLabel?.textAlignment = .center
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
 
-            }
+        if let packs = MGLOfflineStorage.shared.packs {
+            return packs.count
+        } else {
+            return 1
+        }
+    }
 
-            return cell
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        cell = UITableViewCell(style: .subtitle, reuseIdentifier: "cell")
```

Comment:
`dequeueReusableCell(identifier:)` always returns a cell, so re-assigning it to a cell again is unnecessary.